content/en/blog/2019/announcing-istio-client-go/index.md

```diff
@@ -16,7 +16,7 @@ for developers to create controllers and perform Create, Read, Update and Delete
 
 This was a highly requested functionality by many Istio users, as is evident
 from the feature requests on the clients generated by [Aspen Mesh](https://github.com/aspenmesh/istio-client-go)
-and the [Knative project](https://github.com/knative/pkg/tree/master/apis/istio).
+and the [Knative project](https://github.com/knative/pkg).
```

Comment:
```suggestion
and the [Knative project](https://github.com/knative/pkg/tree/7325e901c0768a6d3b1b244ecd021a88d45644a4/apis/istio).
```

Wouldn't a permalink to the Knative Istio client at a commit *before* it was removed  make for sense then linking to an arbitrary `pkg` directory with the code removed?

Reply:
Let's remove this fix, it's been addressed already.